oxidation/crates/parsec_api_types/src/invite.rs

```diff
@@ -251,22 +252,24 @@ impl_transparent_data_format_conversion!(
  * InviteUserConfirmation
  */
 
-parsec_data!("schema/invite_user_confirmation.json");
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(
+    into = "InviteUserConfirmationData",
+    from = "InviteUserConfirmationData"
+)]
+pub struct InviteUserConfirmation {
+    pub device_id: DeviceID,
+    pub device_label: Option<DeviceLabel>,
+    pub human_handle: Option<HumanHandle>,
+    pub profile: UserProfile,
+    pub root_verify_key: VerifyKey,
+}
+
+parsec_data!("schema/invite/invite_user_confirmation.json");
```

Comment:
What's the difference between `InviteUserConfirmation` and `InviteUserConfirmationData`?